### PR TITLE
Message related to missing models improved in DNNImageFeaturizers.

### PR DESCRIPTION
**Pull request by Anipik**
Fixes https://github.com/dotnet/machinelearning/issues/2784

New error msg 
```
[28-02-2019 21:12:14 Error] [xUnit.net 00:00:01.96]     Microsoft.ML.Tests.DnnImageFeaturizerTests.OnnxStatic [FAIL]
[28-02-2019 21:12:14 Informational] [xUnit.net 00:00:01.96]       System.IO.IOException : Model file path C:\git\machinelearning\bin\AnyCPU.Debug\Microsoft.ML.OnnxTransformerTest\netcoreapp2.1\DnnImageModels1\ResNetPrepOnnx\ResNetPreprocess.onnx doesnot exists.
[28-02-2019 21:12:14 Informational] [xUnit.net 00:00:01.96]       Stack Trace:
[28-02-2019 21:12:14 Informational] [xUnit.net 00:00:01.96]         C:\git\machinelearning\src\Microsoft.ML.Core\Utilities\Contracts.cs(744,0): at Microsoft.ML.Contracts.CheckIO(IExceptionContext ctx, Boolean f, String msg, Object[] args)
[28-02-2019 21:12:14 Informational] [xUnit.net 00:00:01.96]         C:\git\machinelearning\src\Microsoft.ML.OnnxTransformer\OnnxTransform.cs(168,0): at Microsoft.ML.Transforms.OnnxTransformer..ctor(IHostEnvironment env, Options options, Byte[] modelBytes)
[28-02-2019 21:12:14 Informational] [xUnit.net 00:00:01.96]         C:\git\machinelearning\src\Microsoft.ML.OnnxTransformer\OnnxTransform.cs(252,0): at Microsoft.ML.Transforms.OnnxTransformer..ctor(IHostEnvironment env, String[] outputColumnNames, String[] inputColumnNames, String modelFile, Nullable`1 gpuDeviceId, Boolean fallbackToCpu)
[28-02-2019 21:12:14 Informational] [xUnit.net 00:00:01.96]         C:\git\machinelearning\src\Microsoft.ML.OnnxTransformer\OnnxTransform.cs(544,0): at Microsoft.ML.Transforms.OnnxScoringEstimator..ctor(IHostEnvironment env, String[] outputColumnNames, String[] inputColumnNames, String modelFile, Nullable`1 gpuDeviceId, Boolean fallbackToCpu)
[28-02-2019 21:12:14 Informational] [xUnit.net 00:00:01.96]         C:\git\machinelearning\src\Microsoft.ML.DnnImageFeaturizer.ResNet1
```

### CODE REVIEW

**Comment by Anipik** [comment]
@TomFinley @Ivanidzo4ka @eerhardt @justinormont can you review this PR ?

File: src/Microsoft.ML.OnnxTransformer/OnnxTransform.cs [hunk 1]
```diff
@@ -165,7 +165,7 @@ private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, Dat
                 if (modelBytes == null)
                 {
                     Host.CheckNonWhiteSpace(options.ModelFile, nameof(options.ModelFile));
-                    Host.CheckUserArg(File.Exists(options.ModelFile), nameof(options.ModelFile));
+                    Host.CheckIO(File.Exists(options.ModelFile), "Model file path {0} doesnot exists.", options.ModelFile);
```
Comment (justinormont) [inline src/Microsoft.ML.OnnxTransformer/OnnxTransform.cs]: Missing space: (also the file could be missing, not just the path to it)
```suggestion
                    Host.CheckIO(File.Exists(options.ModelFile), "Model file {0} does not exists.", options.ModelFile);
```

File: src/Microsoft.ML.Core/Utilities/Contracts.cs [hunk 2]
```diff
@@ -737,7 +737,11 @@ public static void CheckIO(this IExceptionContext ctx, bool f, string msg)
             if (!f)
                 throw ExceptIO(ctx, msg);
         }
-
+        public static void CheckIO(this IExceptionContext ctx, bool f, string msg, params object[] args)
+        {
+            if (!f)
```
Comment (justinormont) [inline src/Microsoft.ML.Core/Utilities/Contracts.cs]: Is the convention to have single letter variables? Seems like golang's style :)

Comment (Anipik) [reply src/Microsoft.ML.Core/Utilities/Contracts.cs]: I am not sure about the convention here, but we are using it for all the contract checks. I think we should keep them same for this pr. we can later modify if we want to

**Pull request merged**